Content.Server/GameTicking/Rules/Components/RevolutionaryRuleComponent.cs

```diff
@@ -0,0 +1,73 @@
+using Content.Shared.Roles;
+using Robust.Shared.Audio;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.GameTicking.Rules.Components;
+
+/// <summary>
+/// Component for the RevolutionaryRuleSystem that stores info about winning/losing, player counts required for starting, as well as prototypes for Revolutionaries and their gear.
+/// </summary>
+[RegisterComponent, Access(typeof(RevolutionaryRuleSystem))]
+public sealed class RevolutionaryRuleComponent : Component
+{
+    /// <summary>
+    /// Stores sessions of Head Revs for end screen.
+    /// </summary>
+    [DataField("headRevs")]
+    public Dictionary<string, string> HeadRevs = new();
```

Comment:
Shouldn't this store mind entities instead.

Reply:
yes its based off nukie copy paste
should store Dictionary<string, Entity>